dotCMS/src/main/java/com/dotcms/security/apps/SecretsKeyStoreHelper.java

```diff
@@ -274,35 +245,33 @@ public synchronized boolean deleteAll(){
      * @throws KeyStoreException
      * @throws Exception
      */
-    @Override
-    public boolean saveValue(final String variableKey, final char[] variableValue) {
+
+    public char [] saveValue(final String variableKey, final char[] variableValue) {
```

Comment:
Codacy found an issue: [Consider using varargs for methods or constructors which take an array the last parameter.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6286472)